src/cow-react/modules/limitOrders/pure/ExecutionPrice/index.tsx

```diff
@@ -0,0 +1,29 @@
+import { TokenAmount } from '@cow/common/pure/TokenAmount'
+import { FiatAmount } from '@cow/common/pure/FiatAmount'
+import { useExecutionPriceFiat } from '@cow/modules/limitOrders/hooks/useExecutionPriceFiat'
+import { Currency, Price } from '@uniswap/sdk-core'
+
+export interface ExecutionPriceProps {
+  executionPrice: Price<Currency, Currency>
+  isInversed: boolean
+}
+
+export function ExecutionPrice({ executionPrice, isInversed }: ExecutionPriceProps) {
+  const executionPriceFiat = useExecutionPriceFiat(executionPrice, isInversed)
+
+  if (!executionPrice) return null
```

Comment:
According to the params, `executionPrice` can't be falsy.

If it can, better make it explicity in the type or if it can't remove this if

Reply:
Thanks! Fixed